tests/test_indexer.py

```diff
@@ -382,6 +383,18 @@ def get_search_results(cls, query, expected_hit_count):
             else:
                 time.sleep(0.5)
 
+    def clear_test_indexes(self):
+        es_client = ElasticsearchClient.get(logger)
+        if es_client.indices.exists(self.dss_index_name):
```

Comment:
under what circumstances would the index not exist at the end of a test?

Reply:
the indexes are created when the first bundle is added. If that process fails, no index will be created.